themes/digital.gov/src/js/gh-edit-commit.js

```diff
@@ -0,0 +1,159 @@
+/**
+ * Sets the most recent commit timestamp for the page
+ * This is displayed in #page-data section at the bottom of the page
+ */
+
+// eslint-disable-next-line func-names
+(function () {
+  const editPageLink = document.querySelector("#page-data .gh-commit-date");
```

Comment:
Curious about the specificity of this selector. Not against it, but couldn't we simply query `.gh-commit-date`? 

Are there multiple elements with this class?

Reply:
Shortened to just one selector, needed to be updated with the new class names.